sdk/python/src/dagger/client/_session.py

```diff
@@ -69,9 +71,17 @@ def __init__(self, conn: ConnectParams, cfg: ConnectConfig | None = None):
             auth=(conn.session_token, ""),
         )
 
+        # test if /schema.json exists, if so, use that as the introspection schema
+        schema_path = Path("/schema.json")
+        introspection = None
+        if schema_path.exists():
+            with schema_path.open() as f:
+                introspection = json.load(f)
+
```

Comment:
This is fine but I think it should be in the CLI rather than the client connection. I'll push a quick fix for that. 👍